.golangci.yml

```diff
@@ -42,6 +43,8 @@ linters-settings:
   ifshort:
     # Maximum length of variable declaration measured in number of characters, after which linter won't suggest using short syntax.
     max-decl-chars: 50
+  gci:
+    local-prefixes: sigs.k8s.io/cluster-api
```

Comment:
Can we use no local prefixes? Most folks out there with goimports won't necessarily make use of it and we can reduce some friction here

Reply:
Done